src/navigator/types.tsx

```diff
@@ -236,10 +236,12 @@ export type StackParamList = {
   [Screens.OutgoingPaymentRequestListScreen]: undefined
   [Screens.PaymentRequestConfirmation]: {
     transactionData: TransactionDataInput
+    isFromScan?: boolean
```

Comment:
seems like it would be safer to make this required. otherwise it seems like we could easily just leave out the optional field, and start getting bad data, because later it is assumed that when `isFromScan` is undefined, it means false.

Reply:
setting it explicitly everywhere seems safer

Reply:
Updated so it's non-optional